packages/hooks/src/useDevice.ts

```diff
@@ -0,0 +1,22 @@
+import { useWindowSize } from 'usehooks-ts';
+
+const BREAKPOINTS = {
+    maxMobile: 600,
+    maxTablet: 1280,
+};
+
+/** A custom hook to check for the client device and determine the layout to be rendered */
+const useDevice = () => {
+    const { width } = useWindowSize();
+    const isMobile = width > 0 && width < BREAKPOINTS.maxMobile;
```

Comment:
Please use snake_case here since it's not a function. Same for the rest.

Reply:
I use camelcase to avoid confusion with is_mobile from ui store

Reply:
I think it's okay to use camelCase (even if you were not trying to avoid confusion) in new files because we will try to migrate to airbnb style guide anyway

Reply:
Why do we want to migrate to airbnb style guide? What we have now and have had since inception of this project works perfectly fine, no matter if it's airbnb or booking.com or whatever. We keep making pointless changes to the codebase that do not improve or impact development process in any positive way. I'd say we should avoid these changes and stick to whatever we already have for a long time and never complained about.

Reply:
@akmal-deriv Unfortunately, since this PR was merged https://github.com/binary-com/deriv-app/pull/11344/files, our README is saying: "Airbnb JavaScript Style Guide is being followed in our code base". Our team agreed to follow it in new files and for new variables.